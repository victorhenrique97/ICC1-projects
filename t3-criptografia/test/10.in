9.bin
1
