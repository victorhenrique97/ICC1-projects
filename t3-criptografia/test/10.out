1788754148149387388893339462889304004106477329146928902630270668886917323245735093544693625930013910649008900065310682255731910689106652364529386938000849046385552000090451119014938089193122086346918749000136919114847733387008625853409339108917936873788574633634353804843431085930841223327543048912139353106391938525426806391669090491891673006769132916133573577736911691516302874619118761873935143743662793916143935069148559128912293377377520025089236433682154509110855911
